### PR TITLE
Fix info on UnconstrainedMeasurementException

### PR DESCRIPTION
**Pull request by tcNickolas**
This change fixes broken exception name and link to its documentation, as well as the reference to its namespace.

### CODE REVIEW

File: articles/machines/qc-trace-simulator/index.md [hunk 1]
```diff
@@ -81,9 +81,10 @@ namespace Quantum.MyProgram
 }
 ```
 
-Note that if there is at least one measurement not annotated using `AssertProb`
-or `ForceMeasure` the simulator will throw `UnconstraintMeasurementException`
-from the `Microsoft.Quantum.Simulation.QCTraceSimulatorRuntime` namespace. See the API documentation on [UnconstraintMeasurementException](https://docs.microsoft.com/dotnet/api/microsoft.quantum.simulation.simulators.qctracesimulators.unconstraintmeasurementexception) for more details.
+Note that if there is at least one measurement not annotated using `AssertProb`,
+the simulator will throw `UnconstrainedMeasurementException`
+from the `Microsoft.Quantum.Simulation.Simulators.QCTraceSimulators` namespace. See the API documentation on 
+[UnconstrainedMeasurementException](https://docs.microsoft.com/dotnet/api/microsoft.quantum.simulation.simulators.qctracesimulators.unconstrainedmeasurementexception) for more details.
```
Comment (cgranade) [inline articles/machines/qc-trace-simulator/index.md]: I think it be better long-term to make this an xref to the .NET API Browser than a URL, as that should persist through changes in how the .NET API Browser is laid out. 
```suggestion
[UnconstrainedMeasurementException](xref:Microsoft.Quantum.Simulation.Simulators.QCTraceSimulators.UnconstrainedMeasurementException) for more details.
```

File: articles/machines/qc-trace-simulator/index.md [hunk 2]
```diff
@@ -100,6 +101,5 @@ Each of these components may be enabled by setting appropriate flags in
 components are provided in the corresponding reference files. See the API documentation on [QCTraceSimulatorConfiguration](https://docs.microsoft.com/dotnet/api/Microsoft.Quantum.Simulation.Simulators.QCTraceSimulators.QCTraceSimulatorConfiguration) for specific details.
 
 ## See also
-The quantum computer [trace simulator
-](https://docs.microsoft.com/dotnet/api/Microsoft.Quantum.Simulation.Simulators.QCTraceSimulators.QCTraceSimulator) C# reference 
+The quantum computer [trace simulator](https://docs.microsoft.com/dotnet/api/Microsoft.Quantum.Simulation.Simulators.QCTraceSimulators.QCTraceSimulator)  C# reference
```
Comment (cgranade) [inline articles/machines/qc-trace-simulator/index.md]:
```suggestion
The quantum computer [trace simulator](xref:Microsoft.Quantum.Simulation.Simulators.QCTraceSimulators.QCTraceSimulator) C# reference 
```

**Comment by natke** [comment]
Updated base branch to master

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/machines/qc-trace-simulator/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/d951ba012e9ac74bd4b87d5b2722a5947ee8f525/articles/machines/qc-trace-simulator/index.md) | 83 | 43 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/26339f51-6d45-4df7-bde2-91b3ec97fe2b_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

**Pull request merged**